catalyst_voices/packages/catalyst_voices_blocs/lib/src/bloc_builder_selector.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+typedef BlocValueSelector<T, S> = T Function(S state);
+
+/// Simple extension on [BlocBuilder] but makes it easy to work with nested
+/// objects when you care only about them.
+class BlocBuilderSelector<B extends StateStreamable<S>, S, T>
```

Comment:
See BlocSelector from flutter_bloc package

Reply:
I think it does exactly what you need 🗡️

Reply:
Oh! Thats convenient!